libraries/client/client_webhook.py

```diff
@@ -187,6 +192,12 @@ def process_webhook(self):
             source_url = self.build_multipart_source(file_key, book)
             identifier, job = self.send_job_request_to_tx_manager(commit_id, source_url, rc, repo_name, repo_owner,
                                                                   count=book_count, part=i, book=book)
+
+            # Send lint request to tx-manager
+            lint_results = self.send_lint_request_to_tx_manager(job, source_url)
```

Comment:
Right, still a WIP. I was trying to figure out the best way to do that. Hadn't come back to that yet. Was going to cross that bridge when I tried this out with your linting code...that is why I need that first before you use what I'm doing. Yet sounds like you need new information as well...hmm.